components/channel_info_modal/channel_info_modal.jsx

```diff
@@ -13,7 +13,20 @@ import PropTypes from 'prop-types';
 
 import React from 'react';
 
-export default class ChannelInfoModal extends React.Component {
+export default class ChannelInfoModal extends React.PureComponent {
+    static propTypes = {
+
+        /**
+         * Function that is called modal is hidden
```

Comment:
Minor grammar mistake `Function that is called modal is hidden` -> `Function that is called when modal is hidden`